helm-charts/whitesource-renovate/templates/deployment.yaml

```diff
@@ -133,6 +133,9 @@ spec:
             {{- end }}
             - name: {{ .Release.Name }}-cache-volume
               mountPath: /tmp/renovate
+          {{- if ne (len .Values.extraVolumeMounts) 0 }}
+{{ toYaml .Values.extraVolumeMounts | indent 12 }}
```

Comment:
This leading whitespace seems wrong, or is it intentional?

Reply:
It's intentional for `indent` but swapped it out for `nindent | trim` which appears to be the preferred way to do this.